ci/README.md

```diff
@@ -0,0 +1,67 @@
+#### What is functional test:
+
+Functional CI is about running functional tests in ci.centos.org for the whole build pipeline
+to check its end to end functionality. This aims to check behavior for all the stages in the 
+pipeline and matches with expected one. If there is any difference in expected and actual status
+of the stages, CI fails and prints all the logs for the pipeline run.
```

Comment:
So if the tests are successful, it doesn't print the logs? If yes, I'd request we be verbose irrespective of success or failure.

Reply:
added logs for both success and fail